src/main/java/com/ashkan/moviesapi/exceptions/NotFound/MovieRentalNotFoundException.java

```diff
@@ -0,0 +1,11 @@
+package com.ashkan.moviesapi.exceptions.NotFound;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(code = HttpStatus.NOT_FOUND)
+public class MovieRentalNotFoundException extends RuntimeException {
+    public MovieRentalNotFoundException(Integer id) {
+        super("Movie Rental with id: '" + id.toString() + "' does not exists!");
```

Comment:
As a good practice instead of use + for join strings you can use String.format(string, param1, param2...)

Reply:
done.